### PR TITLE
Update to qt6

### PR DESCRIPTION
**Pull request by marmarta**
Update all programs to use Qt6. May or may not cause weird palette bugs in dom0 (local testing inconclusive); please test.

Based on https://github.com/QubesOS/qubes-manager/pull/382 

Does not involve any actual changes in functionality (hopefully).

fixes https://github.com/QubesOS/qubes-issues/issues/9476

### CODE REVIEW

**Comment by marmarta** [comment]
@marmarek , could I get this tested in openqa? There are multiple ways this could go wrong, I'd like to see how it looks on a non-frankenstein system.

**Comment by marmarek** [comment]
Sure

**Comment by marmarta** [comment]
openqa worked, in that it found an instant death bug. Asking for another round.

**Comment by marmarta** [comment]
Ready for re-review and tests, @marmarek

**Comment by marmarek** [comment]
- The install template button (and other blue ones) looks inactive
- Error message has all lines joined (see for example list of VMs where a template is used when trying to remove it)
- Abort doesn't actually abort the action, it continues in the background (should the abort button even be there?)

**Comment by marmarek** [comment]
There should be also info what about cloned templates or otherwise not installed with qvm-template initially. Currently they are completely missing on the list, and there is no info there might be some other templates somewhere.

**Comment by marmarek** [comment]
There is updated openQA run, with another set of failures (see updated comment), but also:
```
Traceback (most recent call last):
  File "/usr/bin/qubes-template-manager", line 33, in <module>
    sys.exit(load_entry_point('qubesmanager==4.3.4', 'console_scripts', 'qubes-template-manager')())
             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/lib/python3.13/site-packages/qubesmanager/template_manager.py", line 432, in main
    utils.run_asynchronous(TemplateManagerWindow)
    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/site-packages/qubesmanager/utils.py", line 552, in run_asynchronous
    window = window_class(qt_app, qubes_app, dispatcher)
  File "/usr/lib/python3.13/site-packages/qubesmanager/template_manager.py", line 56, in __init__
    self.prepare_lists()
    ~~~~~~~~~~~~~~~~~~^^
  File "/usr/lib/python3.13/site-packages/qubesmanager/template_manager.py", line 87, in prepare_lists
    self.change_all_combobox.addItem(self.tr('(select template)'))
    ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'TemplateManagerWindow' object has no attribute 'change_all_combobox'
```

**Comment by marmarta** [comment]
another round of changes done; I removed the Abort button for now.

**Comment by marmarta** [comment]
hopefully I fixed the damn async bugs here. I'm going to be chased by coroutines in my nightmares today.

File: ui/restoredlg.ui [hunk 1]
```diff
@@ -225,11 +225,14 @@
     <item>
      <widget class="QTextEdit" name="confirm_text_edit">
       <property name="html">
-       <string>
-&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
+       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
```
Comment (DemiMarie) [inline ui/restoredlg.ui:228]:
```suggestion
       <string>&lt;!DOCTYPE html&gt;
```

The other HTML DOCTYPEs are deprecated.

Comment (marmarta) [reply ui/restoredlg.ui:228]: this is generated by designer-qt, so you can report this upstream. Editing it manually is a bad idea. Same to your comments on https://github.com/QubesOS/qubes-desktop-linux-menu/pull/50

Comment (DemiMarie) [reply ui/restoredlg.ui:228]: I did not realize this was generated code.

Comment (marmarta) [reply ui/restoredlg.ui:228]: while in this file it might not be obvious, the ones in the other PR literally have "generated by XYZ software" in them...

**Comment by marmarta** [comment]
hopefully fixed the last failing test...

**Review by marmarek — APPROVED** [review]

**Pull request merged**